js/Stock/Indicators/IKHIndicator.js

```diff
@@ -6,12 +6,26 @@
  *
```

Comment:
This whole file needs to be deleted and with that the referencing masters files needs to be updated.

Reply:
Deleted file, and updated masters files